chunky/src/java/se/llbit/math/Octree.java

```diff
@@ -539,9 +557,14 @@ public boolean enterBlock(Scene scene, Ray ray, BlockPalette palette) {
         ray.distance += distance;
         distance = 0;
         if (currentBlock.intersect(ray, scene)) {
-          if (prevBlock != currentBlock)
+          if (prevBlock != currentBlock) { //|| (currentBlock.refractive)) {
+            //for things like glass panes coerce the current mat to be air or water
+            // TODO testing
+            if (currentBlock.refractive) {
+              //ray.setCurrentMaterial(currentBlock.waterlogged ? Water.INSTANCE : Air.INSTANCE);
+            }
```

Comment:
Either remove this or comment it alltogether.

```suggestion
            // if (currentBlock.refractive) {
            //    ray.setCurrentMaterial(currentBlock.waterlogged ? Water.INSTANCE : Air.INSTANCE);
            // }
```